bricks/aem-testimonials/aem-testimonial-slide.html

```diff
@@ -0,0 +1,35 @@
+<template>
+  <style>
+    .tabpanel {
+      display: flex;
+    }
+
+    .titles p:first-of-type {
```

Comment:
This doesn't work since the element is in a slot :(

Reply:
Note that you can select a first-level slotted element, `::slotted(p)` works only if the `p` is a child of the `slot`, but not a descendant. That probably doesn't help here.